theme/models.py

```diff
@@ -9,7 +9,7 @@
 from django.db import models
 from django.db.models.signals import pre_save
 from django.template import RequestContext, Template, TemplateSyntaxError
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
also need to replace here?
https://github.com/hydroshare/hydroshare/blob/4738-django-3_2/hs_core/admin.py#L6
https://github.com/hydroshare/hydroshare/blob/4738-django-3_2/theme/forms.py#L4
https://github.com/hydroshare/hydroshare/blob/4738-django-3_2/theme/views.py#L24

Reply:
Yes.